#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("gc_size", double_t, 0, "Cluster size", 0.01, 0.0001, 1.0)
gen.add("gc_thresh", double_t, 0, "Clustering threshold", 5.0, 0.0001, 100.0)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "GeometricConsistencyGrouping"))
